_runners/vault.py

```diff
@@ -73,7 +73,10 @@ def _get_policies(minion_id, config):
 
   policies = []
   for pattern in policyPatterns:
-    policies.append(pattern.format(**mappings))
+    try:
+      policies.append(pattern.format(**mappings))
+    except KeyError:
```

Comment:
Given that #14 is now resolved, do you still see ValueError instead of KeyError?